Allay-Server/src/main/java/org/allaymc/server/block/component/BlockBaseComponentImpl.java

```diff
@@ -83,8 +83,8 @@ public Set<ItemStack> getDrops(BlockStateWithPos blockState, ItemStack usedItem,
     }
 
     @Override
-    public boolean canResetFallDistance() {
-        return blockType.getMaterial().isLiquid();
+    public float getFallDamageReductionFactor() {
+        return 0.0f;
```

Comment:
> Instead of creating a method, I think we can create an property in BlockStateData

Ok. I created new property(not commited yet). But how from where we will get info of reduction factor? There is no such field in block_states.json

Reply:
Added property, but now I still don't know how to change it